tensorflow_ranking/python/model.py

```diff
@@ -56,13 +57,17 @@ def __init__(self, transform_fn=None):
     else:
       self._transform_fn = transform_fn
 
-  def _call_transform_fn(self, features, mode):
+  def _call_transform_fn(self, features, mode, params):
```

Comment:
I just followed the pattern that was here before where it checks if `mode` is defined which caused it to grow into 3 conditionals to cover all the combinations of `{features, mode, params}`...